### PR TITLE
More robust testing for getFileSubmissions

### PR DESCRIPTION
**Pull request by ZacharyRizer**
### Description:

* add failure case for fileSubmissions api call
* create the url string from new URLSearchParams()
* change the name of the test file

Related to ticket: https://cdc-dexops.atlassian.net/browse/PORTAL-1446

### CODE REVIEW

**Review by tbolt — APPROVED** [review]

File: src/mocks/handlers.ts [hunk 1]
```diff
@@ -3,7 +3,16 @@ import API_ENDPOINTS from "src/config/api";
 import mockFileSubmissions from "src/mocks/data/fileStatus.json";
 
 export const handlers = [
-  http.get(API_ENDPOINTS.fileSubmissions, () => {
+  http.get(API_ENDPOINTS.fileSubmissions, ({ request }) => {
+    const url = new URL(request.url);
+    const dataStreamId = url.searchParams.get("data_stream_id");
+
+    // bad request wihtout data_stream_id
```
Comment (sublettk) [inline src/mocks/handlers.ts:10]: Nit: I don't think this comment is necessary. Also, there's a typo in without.

File: src/mocks/handlers.ts [hunk 2]
```diff
@@ -3,7 +3,16 @@ import API_ENDPOINTS from "src/config/api";
...
+    if (!dataStreamId) {
+      return new HttpResponse(null, { status: 400 });
+    }
+
+    // call successful
```
Comment (sublettk) [inline src/mocks/handlers.ts:15]: Nit: I don't think this comment is necessary.

Comment (ZacharyRizer) [reply src/mocks/handlers.ts]: Removed.

File: src/mocks/handlers.ts (hunk 1 above)
Comment (tbolt) [reply src/mocks/handlers.ts]: yea, these 2 comments could come out

**Pull request merged**